missing crypt32.lib in Makefile.win

**Problem details**

Need the same crypt32 lib in Makefile.win as was added for CMake in 5d30d8efa342601d20faf88ba6b10922d75f5a01

Otherwise we can get `error LNK2019: unresolved external symbol __imp__CertOpenStore@20 referenced in function _capi_open_store` etc.

**Steps/codes to reproduce the bug**

1. Using https://raw.githubusercontent.com/fiendish/lua-openssl-build/master/appveyor.yml  (my custom changes for static linking the DLL)

**fix**
2. This change fixes the build: https://github.com/fiendish/lua-openssl/commit/ec19d12f2e789abc1acfe5dd4dcc4c3b15f7d77d#diff-a2f2a3db29e468ef4d80b60ae2ee1b4e

--- a/Makefile.win
+++ b/Makefile.win
@@ -19,7 +19,7 @@ lib: src\$T.dll
 
 src\$T.dll: $(OBJS)
 	link /DLL /out:src\$T.dll $(OBJS) "$(LUA_LIB)" "$(OPENSSL_LIB)" \
-		ws2_32.lib kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib 
+		crypt32.lib ws2_32.lib kernel32.lib user32.lib gdi32.lib winspool.lib comdlg32.lib advapi32.lib 
 	IF EXIST src\$T.dll.manifest mt -manifest src\$T.dll.manifest -outputresource:src\$T.dll;2
 
 install: src\$T.dll

